packages/plus/package.json

```diff
@@ -43,20 +44,21 @@
   "devDependencies": {
     "@babel/core": "7.23.2",
     "@emotion/babel-plugin": "11.11.0",
+    "@emotion/react": "11.11.1",
+    "@emotion/styled": "11.11.0",
```

Comment:
not the same version as `@emotion/react` ?

Reply:
@philibea this is the last version available for `@emotion/styled`
> Source: https://www.npmjs.com/package/@emotion/styled?activeTab=versions